lang/cs/Org.Apache.REEF.Evaluator/Evaluator.cs

```diff
@@ -81,8 +82,8 @@ public static void Main(string[] args)
                 var serializer = injector.GetInstance<AvroConfigurationSerializer>();
                 var rootEvaluatorConfiguration = 
                     TangFactory.GetTang().NewConfigurationBuilder(serializer.FromString(injector.GetNamedInstance<EvaluatorConfiguration, string>()))
-                        .BindSetEntry<IClock.RuntimeStartHandler, EvaluatorRuntime, IObserver<RuntimeStart>>()
-                        .BindSetEntry<IClock.RuntimeStopHandler, EvaluatorRuntime, IObserver<RuntimeStop>>()
+                        .BindSetEntry<RuntimeStartHandler, EvaluatorRuntime, IObserver<RuntimeStart>>()
```

Comment:
I believe we still need to support `IClock.RuntimeStartHandler` to be truly backward compatible.